jetty-core/jetty-quic/jetty-quic-server/src/main/java/org/eclipse/jetty/quic/server/ServerQuicSession.java

```diff
@@ -44,12 +45,14 @@
 public class ServerQuicSession extends QuicSession implements CyclicTimeouts.Expirable
 {
     private final Connector connector;
+    private final ServerQuicConfiguration quicConfiguration;
```

Comment:
This field is already present in `ServerQuicConnection`, so I would not duplicate it.